tools/newRepo

```diff
@@ -19,56 +19,59 @@
 //  VP   V8P Y88888P  `8b8' `8d8'  88   YD Y88888P 88       `Y88P'
 
 const inquirer = require("inquirer");
-const client = require("../src/client.js"); 
+const client = require("../src/client.js");
```

Comment:
Ahhh, got it. It's because it doesn't end in `.js`, which is the requirement in the gulpfile for files passed to eslint. I'm going to see how can we address that...